### DISCUSSION TITLE
HEIC files are ignored during indexing

### DISCUSSION THREAD

**Discussion by kmorber** [photoprism/photoprism#3707] [Q&A]
Hello,

HEIC-files are missing from photostructure. In log i can see the following message:

time="2023-09-04T21:25:09Z" level=info msg="index: ignored \"2021/07/31/20210731_194749.HEIC\""
time="2023-09-04T21:25:09Z" level=info msg="index: ignored \"2021/08/03/20210803_053034.HEIC\""
time="2023-09-04T21:25:09Z" level=info msg="index: ignored \"2021/08/03/20210803_053037.HEIC\""
time="2023-09-04T21:25:09Z" level=info msg="index: ignored \"2021/08/07/20210807_214229.HEIC\""
time="2023-09-04T21:25:09Z" level=info msg="index: ignored \"2021/08/07/20210807_214347.HEIC\""
time="2023-09-04T21:25:09Z" level=info msg="index: ignored \"2021/08/08/20210808_110745.HEIC\""
time="2023-09-04T21:25:09Z" level=info msg="index: ignored \"2021/08/08/20210808_204512.HEIC\""
time="2023-09-04T21:25:09Z" level=info msg="index: ignored \"2021/08/08/20210808_204513.HEIC\""
time="2023-09-04T21:25:09Z" level=info msg="index: ignored \"2021/08/09/20210809_150253.HEIC\""
time="2023-09-04T21:25:09Z" level=info msg="index: ignored \"2021/08/09/20210809_200431.HEIC\""

What am i doing wrong?

BR

Karsten
[log.txt](https://github.com/photoprism/photoprism/files/12516886/log.txt)

**Comment by graciousgrey** [comment]
Hey :wave: 

Is it possible that you have a .ppignore file configured in a way that excludes heic files?

**Comment by kmorber** [comment]
An other problem is, that /scripts/build-libheif.sh is no more working, because v1.16.2 is reported by the script, but cannot be found on [dl.photoprism](https://dl.photoprism.app/dist/libheif/).

**Reply by graciousgrey** [reply]
Unless you are using our photoprism/develop Docker image to build PhotoPrism from source, missing build dependencies such as libheif need to be installed first, as shown in the Dockerfile on which our official development environment is based. Since you often don't need to use the exact same versions, you can replace many of the packages with those available on your operating system.
If the pre-built libheif versions from our download server are not suitable for your environment, you will need to manually install it. We recommend finding an installable package for this, as building it from source is a very complicated process that can take many hours

**Reply by kmorber** [reply]
Sorry for my late response,

it has been solved, there was a .ppignore file which excluded heif files.
